### PR TITLE
Add Git Foundations Course by GitKraken - Fix

### PR DESCRIPTION
**Pull request by stmapman7**
## What does this PR do?
**Add resource(s)** | Remove resource(s) | Add info | Improve repo

### Description
This is a free Git foundations course from Gitkraken, a proprietary project management Git tool for developers. 

### Why is this valuable (or not)?
It's another free learning Git course for beginners. 

### How do we know it's really free?
You will need to create an account using your e-mail but after signing up you are taken directly to the course. 

### For book lists, is it a book? For course lists, is it a course? etc.
This is a course and there is only one course at this time. Others may be created soon. 

## Checklist:
- [X] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [X] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [X] Include author(s) and platform where appropriate.
- [X] Put lists in alphabetical order, correct spacing.
- [X] Add needed indications (PDF, access notes, under construction).
- [X] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by shrn01** [comment]
Looks like you have taken a wrong branch again. I would suggest you delete the fork, create a fresh fork and make your change on top of it in the main branch and raise a PR.

File: .github/workflows/check-urls.yml [hunk 1]
```diff
@@ -40,7 +40,7 @@ jobs:
 
       - name: Get changed files
         id: changed-files
-        uses: tj-actions/changed-files@v30.0.0
+        uses: tj-actions/changed-files@v32.1.2
```
Comment (shrn01) [inline .github/workflows/check-urls.yml:43]: You can alternatively remove this change and update the PR

Comment (stmapman7) [reply .github/workflows/check-urls.yml:43]: > "Looks like you have taken a wrong branch again. I would suggest you delete the fork, create a fresh fork and make your change on top of it in the main branch and raise a PR."
This is the main branch. I will only delete my fork if other reviewers on this repo recommend ths. 

> "You can alternatively remove this change and update the PR." 
Checking to see if there's a way to revert this. It's noted this change from a script from other contributors here.

**Comment by eshellman** [comment]
don't change the workflows in a content PR!!!

**Comment by stmapman7** [comment]
Doing as suggested. Closing PR

File: .github/workflows/check-urls.yml (hunk 1 above)
Comment (stmapman7) [reply .github/workflows/check-urls.yml:43]: Resolved #8748

**Pull request closed without merging**